lib/index.js

```diff
@@ -52,6 +52,9 @@ function exec(command, ...args) {
   return deferred.promise;
 }
 
+const endsWith = (suffix, str) =>
+  str.indexOf(suffix, str.length - suffix.length) !== -1;
+
```

Comment:
`Object .css has no method 'endsWith'` otherwise, not even sure it's about core-js.